src/main.ts

```diff
@@ -967,6 +970,11 @@ function generateService(
 
   let index = 0;
   for (const methodDesc of serviceDesc.method) {
+
+    if (options.nestJs) {
```

Comment:
This is where I change the case of the methods to lowercase

Reply:
Nice, I ended up adding a separate `lowerCaseServiceNames` flag that `nestJs` changes to `true`.

Reply:
Great!